python/tests/core/ecl/test_grid_statoil.py

```diff
@@ -383,3 +383,7 @@ def test_volume_kw(self):
 
                               
         
+    def test_large_case(self):
+        grdecl_file = self.createTestPath("Statoil/ECLIPSE/1.6.0_issueGrdecl/test_aug2016_gridOnly.grdecl")
+        grid = EclGrid.loadFromGrdecl( grdecl_file )
```

Comment:
What does this test test? That a too big grid is not causing a crash? Or that it actually loads? No asserts...